demos/common/processing/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2018-2019 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+find_package(OpenCV REQUIRED COMPONENTS core imgproc)
+
+FILE(GLOB SOURCES ./src/*.cpp)
+FILE(GLOB HEADERS ./include/*.h)
+
+# Create named folders for the sources within the .vcproj
+# Empty name lists them directly under the .vcproj
+source_group("src" FILES ${SOURCES})
+source_group("include" FILES ${HEADERS})
+
+add_library(processing STATIC ${SOURCES} ${HEADERS})
+target_include_directories(processing PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include" "${CMAKE_CURRENT_SOURCE_DIR}/../include")
```

Comment:
Why is the second directory there? That doesn't belong to this library.

Reply:
because we need some headers (performance_metrics.hpp) from common library

Reply:
Then `processing` should link to `common`.

Reply:
OK. done